server/src/interfaces.ts

```diff
@@ -31,3 +31,22 @@ export interface CustomerPayload {
 export interface CustomerResponse extends CustomerPayload {
   id: number;
 }
+
+/**
+ * MerchantPayload Interface that contains the fields of the payload that
+ * would be sent to create a Merchant Entity.
+ */
+export interface MerchantPayload {
+  name: string;
+  email: string;
+  vpa: string;
+  uid: string;
```

Comment:
What is this?

Reply:
This is the unique ID of the user from Firebase Authentication. Actually, now that I think of it, maybe I should rename it to `firebaseUid`.